src/widgets/histogram/chart.js

```diff
@@ -1499,6 +1501,9 @@ module.exports = cdb.core.View.extend({
       })
       .attr('y', self.chartHeight())
       .attr('height', 0)
+      .attr('data-tooltip', function (d) {
+        return self._tooltipFormatter(d.freq);
+      })
```

Comment:
`this.formatter` depends on the column type. For tooltips, we always need to format as a number.